modules/persistent-storage-csi-efs-metrics-overview.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+// 
+
+[id="efs-metrics-overview_{context}"]
+= Usage metrics overview
+
+Amazon Web Services (AWS) Elastic File Service (EFS) usage metrics allows you to monitor how much EFS space is used by the {product-title}.
+
+[IMPORTANT]
+====
+This features is disabled by default, since turning on metrics can lead to performance degradation.
```

Comment:
ISG: For translation, use "since" only for time, not as a synonym for "because"
```suggestion
This features is disabled by default, because turning on metrics can lead to performance degradation.
```